src/components/Notification/Notification.tsx

```diff
@@ -4,7 +4,7 @@ import {
   useNotificationListQuery,
   useNotificationPost,
 } from '@/api/services/notification/quries'
-import { useState } from 'react'
+import { useState, useEffect, useRef } from 'react'
```

Comment:
```suggestion
import React, { useState, useEffect, useRef } from 'react'
```
[P1] 의 import 추가 입니다.